projects.html

```diff
@@ -0,0 +1,35 @@
+---
+title: Projects
+permalink: /projects/
+description: >-
+  Open Free Energy develops and maintains several software tools used in
+  alchemical free energy calculations.
+---
+
+<table class="projects">
+  {% assign roles = "flagship developers maintainers" | split: " " %}
+  {% for role in roles %}
+  {% for project in site.projects %}
```

Comment:
can we hack the jinja here to list openfe at the top of the list?

Reply:
Already done, as long as openfe remains the only thing marked as a "flagship" project.